src/tree/EnvironmentTreeItem.ts

```diff
@@ -169,8 +168,7 @@ export class EnvironmentTreeItem extends AzureParentTreeItem implements IAzureRe
         const client: WebSiteManagementClient = await createWebSiteClient(this.root);
         this.data = await client.staticSites.getStaticSiteBuild(this.parent.resourceGroup, this.parent.name, this.buildId);
 
-        this.localProjectPath = getSingleRootFsPath();
```

Comment:
This has to be set somewhere in `EnvironmentTreeItem` because `GitHubConfigGroupTreeItem` depends on it